models/volume_transmitter.h

```diff
@@ -48,23 +48,25 @@ Description
 +++++++++++
 
 The volume transmitter is used in combination with neuromodulated
-synaptic plasticty, plasticity that depends not only on the activity
+synaptic plasticity, plasticity that depends not only on the activity
 of the pre- and the postsynaptic neuron but also on a non-local
 neuromodulatory third signal. It collects the spikes from all neurons
-connected to the volume transmitter and delivers the spikes to a
-user-specific subset of synapses.  It is assumed that the
-neuromodulatory signal is a function of the spike times of all spikes
-emitted by the population of neurons connected to the volume
-transmitter.  The neuromodulatory dynamics is calculated in the
-synapses itself. The volume transmitter interacts in a hybrid
-structure with the neuromodulated synapses. In addition to the
-delivery of the neuromodulatory spikes triggered by every pre-synaptic
-spike, the neuromodulatory spike history is delivered in discrete time
-intervals of a manifold of the minimal synaptic delay. In order to
-insure the link between the neuromodulatory synapses and the volume
-transmitter, the volume transmitter is passed as a parameter when a
-neuromodulatory synapse is defined. The implementation is based on the
-framework presented in [1]_.
+connected to the volume transmitter and delivers the spikes to a subset
+of synapses in the network. The user specifies this subset by
+passing the volume transmitter as a parameter when a
+neuromodulatory synapse is defined.
+
+It is assumed that the neuromodulatory signal is a function of the
+spike times of all spikes emitted by the population of neurons connected
+to the volume transmitter. The neuromodulatory dynamics is calculated
+in the synapses itself.
+
+The volume transmitter interacts in a hybrid structure with the neuromodulated
+synapses: In addition to the delivery of the neuromodulatory spikes triggered
+by every pre-synaptic spike, the neuromodulatory spike history is delivered
+in discrete time intervals of a manifold of the minimal synaptic delay.
```

Comment:
"manifold" seems like not the right word to use here.

Reply:
I changed it, but would like to hear explicitly that I didn't misunderstand the previous formulation and what is now written there is correct.